### PR TITLE
Add narrative to the edit/show page of referral

### PR DESCRIPTION
**Pull request by sumancc**
[#127202073]

### CODE REVIEW

File: app/models/referral.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@ class Referral # :nodoc:
     :name,
     :response_time,
     :screening_decision,
-    :started_at
```
Comment (nate-cc) [inline app/models/referral.rb]: these should be sorted

**Comment by nate-cc** [comment]
See comments, other than that all good

**Comment by nate-cc** [comment]
Do you think we need a controller test to make sure that strong params is setup correctly?

**Comment by sumancc** [comment]
We are using strong params like a dsl so I don't know if a controller test makes sense. What about strong params would you want to test?

**Pull request closed without merging**